pkg/mpaland-printf/patches/0002-RIOT-integration.patch

```diff
@@ -30,7 +30,7 @@ index 8a700ad..4a17672 100644
 +/* RIOT integration: Use stdio_write for output */
 +static void _putchar(char character)
 +{
-+    stdio_write(&character, sizeof(character));
++  stdio_write(&character, sizeof(character));
```

Comment:
Looking at a patch of a patch in a browser :smile:  Was this white-space edit intentional? If I am counting my spaces correctly, I assume it is a removal of an extra space that crept into the previous PR?

Reply:
When I changed the patches anyway, I fixed the inconsistent indent. (mplaland-printf uses two spaces for indent instead of the four we use. And out of habit I used RIOT's style, causing a style mess :D)